.travis.yml

```diff
@@ -41,6 +42,8 @@ matrix:
       env: SCALA_VERSION=2.12.0-M2
     - jdk: openjdk6
       env: SCALA_VERSION=2.12.0-M3
+    - jdk: openjdk6
+      env: SCALA_VERSION=2.12.0-M4
```

Comment:
Yes, this entry *excludes* the combination of Java 6 and Scala 2.12.0-M4 from the build matrix. I wonder why you even bother cross-building the 2.11 stuff on Java 8. I'd build 2.11 on Java 6 and 2.12 on Java 8 to keep it simple.